Makefile

```diff
@@ -0,0 +1,7 @@
+IGVM_FILE ?= target/release/migtd.igvm
```

Comment:
Is this new Makefile required to build igvm format migtd? Will it impact default build using "cargo image"?